daprdocs/content/en/developing-applications/building-blocks/service-invocation/howto-invoke-services-grpc.md

```diff
@@ -0,0 +1,238 @@
+---
+type: docs
+title: "How-To: Invoke services using gRPC"
+linkTitle: "How-To: Invoke services using gRPC"
+description: "Call between services using service invocation"
+weight: 3000
+---
+
+This article describe how to deploy services each with an unique application ID, so that other services can discover and call endpoints on them using service invocation API.
```

Comment:
How does this article relate to https://docs.dapr.io/operations/configuration/grpc/#configuring-dapr-to-communicate-with-an-app-via-grpc which refers to using "dapr run --app-protocol grpc"? When should I use this setting in this scenario? 

How does this relate ? "In addition to calling Dapr via gRPC, Dapr can communicate with an application via gRPC. To do that, the app needs to host a gRPC server and implements the Dapr appcallback service"

Reply:
This article doesn't mention "gRPC interface" at all. Again, gRPC proxying and other gRPC articles do not relate at all. There is no connection whatsoever between any of them.

There's an example of when --app-protocol grpc in this sample, its needed when your app needs to get invoked over gRPC.